File: OPENAPI-REST-API/swagger-client/dart/lib/model/inline_response20027.dart
part of swagger.api;

class InlineResponse20027 {
  
  List<InlineResponse20027Bandwidth> bandwidth = [];

  List<InlineResponse20027Ips> ips = [];

  List<InlineResponse20027Os> os = [];

  List<InlineResponse20027Cp> cp = [];

  List<InlineResponse20027Raid> raid = [];

  InlineResponse20027();

  @override
  String toString() {
    return 'InlineResponse20027[bandwidth=$bandwidth, ips=$ips, os=$os, cp=$cp, raid=$raid, ]';
  }

  InlineResponse20027.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    bandwidth = InlineResponse20027Bandwidth.listFromJson(json['bandwidth']);
    ips = InlineResponse20027Ips.listFromJson(json['ips']);
    os = InlineResponse20027Os.listFromJson(json['os']);
    cp = InlineResponse20027Cp.listFromJson(json['cp']);
    raid = InlineResponse20027Raid.listFromJson(json['raid']);
  }

  Map<String, dynamic> toJson() {
    return {
      'bandwidth': bandwidth,
      'ips': ips,
      'os': os,
      'cp': cp,
      'raid': raid
     };
  }

  static List<InlineResponse20027> listFromJson(List<dynamic> json) {
    return json == null ? new List<InlineResponse20027>() : json.map((value) => new InlineResponse20027.fromJson(value)).toList();
  }

  static Map<String, InlineResponse20027> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, InlineResponse20027>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new InlineResponse20027.fromJson(value));
    }
    return map;
  }
}

File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/InlineResponse20027.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class InlineResponse20027 {
    /// <summary>
    /// Gets or Sets Bandwidth
    /// </summary>
    [DataMember(Name="bandwidth", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "bandwidth")]
    public List<InlineResponse20027Bandwidth> Bandwidth { get; set; }

    /// <summary>
    /// Gets or Sets Ips
    /// </summary>
    [DataMember(Name="ips", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "ips")]
    public List<InlineResponse20027Ips> Ips { get; set; }

    /// <summary>
    /// Gets or Sets Os
    /// </summary>
    [DataMember(Name="os", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "os")]
    public List<InlineResponse20027Os> Os { get; set; }

    /// <summary>
    /// Gets or Sets Cp
    /// </summary>
    [DataMember(Name="cp", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "cp")]
    public List<InlineResponse20027Cp> Cp { get; set; }

    /// <summary>
    /// Gets or Sets Raid
    /// </summary>
    [DataMember(Name="raid", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "raid")]
    public List<InlineResponse20027Raid> Raid { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class InlineResponse20027 {\n");
      sb.Append("  Bandwidth: ").Append(Bandwidth).Append("\n");
      sb.Append("  Ips: ").Append(Ips).Append("\n");
      sb.Append("  Os: ").Append(Os).Append("\n");
      sb.Append("  Cp: ").Append(Cp).Append("\n");
      sb.Append("  Raid: ").Append(Raid).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
